### PR TITLE
Add support for withdrawn advisories from Github Api

### PR DESCRIPTION
**Pull request by nick-smit**
See [issue #87](https://github.com/Roave/SecurityAdvisories/issues/87) at Roave/SecurityAdvisories

### CODE REVIEW

File: src/Roave/SecurityAdvisories/AdvisorySources/GetAdvisoriesFromGithubApi.php [hunk 1]
```diff
@@ -80,6 +83,11 @@ public function __invoke(): Generator
             $versions = Type\shape([0 => Type\non_empty_string(), 1 => Type\optional(Type\non_empty_string())])
                 ->assert(Str\split($item['node']['vulnerableVersionRange'], ','));
 
+            if ($item['node']['advisory']['withdrawnAt'] !== null) {
```
Comment (Ocramius) [inline src/Roave/SecurityAdvisories/AdvisorySources/GetAdvisoriesFromGithubApi.php:86]: The return type of `getAdvisories()` needs to be refined a bit: 

```
Cannot access value on variable $item['node'] using offset value of 'advisory', expecting 'vulnerableVersionRange' or 'package' (see https://psalm.dev/115)
            if ($item['node']['advisory']['withdrawnAt'] !== null) {
```

Code all good, IMO - just docblock additions

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: src/Roave/SecurityAdvisories/AdvisorySources/GetAdvisoriesFromGithubApi.php (hunk 1 above)
Comment (nick-smit) [reply src/Roave/SecurityAdvisories/AdvisorySources/GetAdvisoriesFromGithubApi.php:86]: I've resolved the issue in the latest commit.

**Review by Ocramius — APPROVED** [review]
Thanks @nick-smit!

**Comment by Ocramius** [comment]
Ref: https://github.com/Roave/SecurityAdvisories/commit/8a0da9e2c9a96425395e0af80bd3aece26576245

**Pull request merged**